src/config/constants.ts

```diff
@@ -268,3 +268,6 @@ export const DEFAULT_LINK_SHOW_IFRAME = false;
 export const DEFAULT_LINK_SHOW_BUTTON = true;
 
 export const GRAASP_ASSETS_URL = process.env.REACT_APP_GRAASP_ASSETS_URL;
+
+export const TUTORIALS_LINK =
+  'https://player.graasp.org/9d80d81f-ec9d-4bfb-836a-1c6b125aef2f';
```

Comment:
It may make sense to have a dedicated endpoint (e.g. `/tutorials`) so that we can dymanically adjust e.g. in the DB at some point.